components/producers/producer.go

```diff
@@ -28,26 +31,26 @@ var (
 	OutFile string
 	// Append flag will append to the outfile instead of overwriting, useful when there's multiple inresults.
 	Append bool
+	// Debug flag initializes the logger with a debug level
+	Debug bool
 )
 
 const (
 	sourceDir = "/workspace/output"
-
-	// EnvDraconStartTime Start Time of Dracon Scan in RFC3339.
```

Comment:
why where these consts moved? they don't seem to have any influence related to the logging stuff

Reply:
because  they were duplicates, not necessarily part of THIS PR but bear with me, it's tiny lets merge it